docs/docs/maintaining-a-plugin.md

```diff
@@ -1,10 +1,50 @@
 ---
 title: Maintaining a Plugin
-issue: https://github.com/gatsbyjs/gatsby/issues/21067
 ---
 
-This is a stub article meant to be filled with tips on how to maintain a Gatsby plugin once you've published it as an npm package.
+The Gatsby community thrives on the power of plugins, you often find that there is a plugin for almost everything ranging from source plugins to transformer plugins to plugins that are plugins for other plugin . There is a guide to for the [naming convention for plugins](https://www.gatsbyjs.org/docs/naming-a-plugin/) check it out to learn more .
 
-Topics to be covered:
+> Wondering how to [Creating a plugin](https://www.gatsbyjs.org/docs/creating-plugins/) ? Look no further . Start contributing .
 
-- yarn workspaces can solve yarn link inconsistencies
+## What is a Plugin ?
```

Comment:
```suggestion
## What is a plugin?
```

This is a small little thing from the style guide, in subtitles `##`, or `###`, etc, we generally only capitalize the first word: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#format-titles-and-headers